frontend/.env.example

```diff
@@ -50,7 +50,7 @@ AUTH_RASCL_LOGOUT_URL=http://localhost:3000/
 
 # Application feature flags -- see ./app/utils/env.server.ts for valid values
 # (optional; default: "")
-ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,authenticated-status-check, update-governmental-benefit,dependent-status-checker
+ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,authenticated-status-check, update-governmental-benefit,dependent-status-checker,view-playload
```

Comment:
I know this isn't a part of your PR, but can you remove the space before `update-governmental-benefit`, please. I don't think it should be there.

Reply:
removed